overlord/snapstate/snapstate.go

```diff
@@ -1071,7 +1071,7 @@ func applyAutoAliasesDelta(st *state.State, delta map[string][]string, op string
 		msg = i18n.G("Prune automatic aliases for snap %q")
 	}
 	for instanceName, aliases := range delta {
-		if err := CheckChangeConflict(st, instanceName, nil); err != nil {
+		if err := checkChangeConflictIgnoringOneChange(st, instanceName, nil, fromChange); err != nil {
```

Comment:
fascinating how the `fromChange` pattern comes up again

Reply:
yes, these changes (places not taking into account formChange) might even be fixing the issue we saw with check-refresh but we need to dig/double check